gpkit/nomials.py

```diff
@@ -430,14 +449,30 @@ def __float__(self):
 
 
 class Posynomial(Signomial):
+    """A Signomial with strictly positive cs
+
+    Arguments
+    ---------
+    Same as Signomial.
+    Note: Posynomial historically supported several different init formats
+          These will be deprecated in the future, replaced with a single
+          __init__ syntax, same as Signomial.
+    """
     pass
 
 
 class Monomial(Posynomial):
-    '''
-    TODO: Add docstring
-    '''
+    """A Posynomial with only one term
+
+    Arguments
+    ---------
+    Same as Signomial.
+    Note: Monomial historically supported several different init formats
+          These will be deprecated in the future, replaced with a single
+          __init__ syntax, same as Signomial.
+    """
     def __rdiv__(self, other):
+        """Divide other by this Monomial"""
```

Comment:
Unfortunately it's slightly tricky, but got this working in 252197dcbafdfcca4750fdca88c1956fba83ad3d.

The trickiness is that Python only calls `__rdiv__` for two objects of different type. For two objects of the same type, `__rdiv__` is never called. This means that `__div__` needs to support Monomial-Monomial division, for example. The solution I chose was to simply call `__rdiv__` from `__div__` for these cases.